extensions/cl_intel_unified_shared_memory.asciidoc

```diff
@@ -52,7 +52,7 @@ Shipping
 == Version
 
 Built On: {docdate} +
-Revision: 1.0.0
+Revision: 1.1.0
```

Comment:
Since we're bumping the revision, this is a change to set `revision` in the XML https://github.com/KhronosGroup/OpenCL-Docs/blob/main/xml/cl.xml#L6699